improver/wxcode/wxcode_decision_tree_global.py

```diff
@@ -53,13 +53,9 @@ def wxcode_decision_tree_global():
               conditions should be combined;
               e.g. rainfall > 0.5 AND snowfall > 0.5
         * diagnostics_fields: The diagnostics which are being used in the
-              query. If this is a list of lists, the two fields in a given
-              list are subtracted (1st - (2nd * gamma)) and then compared
+              query. If this is a list of lists, the fields are manipulated
+              according to the operators in the list and then compared
               with the probability threshold.
-        * diagnostic_gamma (NOT UNIVERSAL): This is the gamma factor that
-              is used when comparing two fields directly, rather than
-              comparing a single field to a probability threshold.
-              e.g. gamma * P(SnowfallRate) < P(RainfallRate).
```

Comment:
As we don't define these gammas anymore, the ```0.7``` value seems to appear without explanation in various places. Could you document them somewhere, either this doc-string or with inline comments near to where they occur, just to explain what they are doing. The original explanation that's been removed wasn't great anyway, not really explaining their purpose, so there is room to improve.

Reply:
Doc-string updated

Reply:
Thanks Stephen.